tests/test_actions.py

```diff
@@ -971,3 +972,94 @@ def perform_as(self, the_actor: Actor) -> None:
         # kinked microphone will flush the logs back up into the previous
         # kink, which will then be cleared by the outer Silently.
         assert [r.msg for r in caplog.records] == []
+
+
+class TestTryTo:
+    def test_can_be_instantiated(self) -> None:
+        t1 = TryTo()
+        t2 = TryTo(FakeAction()).or_(FakeAction())
+
+        assert isinstance(t1, TryTo)
+        assert isinstance(t2, TryTo)
+
+    def test_implements_protocol(self) -> None:
+        t = TryTo()
+
+        assert isinstance(t, Performable)
+        assert isinstance(t, Describable)
+
+    def test_describe(self) -> None:
+        mock_action1 = FakeAction()
+        mock_action1.describe.return_value = "Be!"
+
+        mock_action2 = FakeAction()
+        mock_action2.describe.return_value = "not to be!"
+    
+        t = TryTo(mock_action1).or_(mock_action2)
+        assert (t.describe() == "TryTo be or not to be")
+
+    def test_first_action_passes(self, Tester, mocker: MockerFixture) -> None:
+        mock_clear = mocker.spy(the_narrator, "clear_backup")
+        mock_flush = mocker.spy(the_narrator, "flush_backup")
+        mock_kink = mocker.spy(the_narrator, "mic_cable_kinked")
+        
+        action1 = FakeAction()
+        action2 = FakeAction()
+        TryTo(action1).or_(action2).perform_as(Tester)
+
+        assert action1.perform_as.call_count == 1
+        assert action2.perform_as.call_count == 0
+
+        assert mock_kink.call_count == 1
+        assert mock_clear.call_count == 1
+        assert mock_flush.call_count == 1
+
+    def test_first_action_fails(self, Tester, mocker: MockerFixture) -> None:
+        mock_clear = mocker.spy(the_narrator, "clear_backup")
+        mock_flush = mocker.spy(the_narrator, "flush_backup")
+        mock_kink = mocker.spy(the_narrator, "mic_cable_kinked")
+        
+        exc = AssertionError("Wrong!")
+        action1 = FakeAction()
+        action2 = FakeAction()
+        action1.perform_as.side_effect = exc
+        
+        TryTo(action1).or_(action2).perform_as(Tester)
+
+        assert action1.perform_as.call_count == 1
+        assert action2.perform_as.call_count == 1
+
+        assert mock_kink.call_count == 1
+        assert mock_clear.call_count == 2
+        assert mock_flush.call_count == 1
+
+    def test_output_first_fails(self, Tester, caplog):
+        
+        class FakeActionFail(Performable):
+            @beat("{} tries to FakeActionFail")
+            def perform_as(self, actor: Actor):
+                raise AssertionError("This Fails!")
+
+        class FakeActionPass(Performable):
+            @beat("{} tries to FakeActionPass")
+            def perform_as(self, actor: Actor):
+                return
+
+        with caplog.at_level(logging.INFO):
+            TryTo(FakeActionFail()).or_(FakeActionPass()).perform_as(Tester)
+        assert caplog.records[0].message == "Tester tries to FakeActionPass"
```

Comment:
Second nitpick: there should be a blank line above this `assert`, to separate the "assert" from the "act" (and same with the next test).